projects/pap/src/app/app.component.ts

```diff
@@ -146,6 +152,39 @@ export class AppComponent {
       .subscribe(_ => {
         this._navCtrl.navigateForward('settings/address');
       });
+    this.isLogged$
+      .pipe(
+        filter(l => l),
+        switchMap(() => this.missedFields$),
+        filter(f => f != null && f.length > 0),
+        switchMap(fields =>
+          this._modalCtrl.create({
+            component: MissedFieldsUserModal,
+            componentProps: {fields},
+          }),
+        ),
+        switchMap(modal => {
+          modal.present();
+          return modal.onWillDismiss();
+        }),
+        switchMap(res => {
+          const SUCESSFULLY_UPDATE: AlertOptions = {
+            cssClass: 'pap-alert',
+            header: 'Aggiornamento avvenuto con successo',
+            message: 'i dati relativi al tuo profilo sono stati aggiornati',
+            buttons: [
+              {
+                text: 'ok',
+                role: 'ok',
+              },
+            ],
+          };
+          return this._alertCtrl.create(SUCESSFULLY_UPDATE);
+        }),
+      )
+      .subscribe(async alert => {
+        (await alert).present();
```

Comment:
niente await async continua il flusso in pipe
aggiungendo una switchMap(alert => alert).subscribe(a=>a.present())